packages/primitives/radio/src/Root.tsx

```diff
@@ -30,27 +39,24 @@ export const RadioInput = component(
   accessibilityLabel,
   value,
   isDisabled,
-  onChange,
+  isVisible,
 }) => (
-  <input
-    type="radio"
-    id={id}
-    name={groupName}
-    key={key || groupName + id}
-    checked={groupValue === value}
-    value={value}
-    aria-labelledby={accessibilityLabelBy.length > 0 ? accessibilityLabelBy.join(' ') : undefined}
-    aria-label={accessibilityLabel}
-    disabled={isDisabled}
-    style={styles}
-    onChange={(evt) => {
-      setGroupValue(evt.currentTarget.value)
-
-      if (typeof onChange === 'function') {
-        onChange(evt)
-      }
-    }}
-  />
+  <label>
+    {children}
+    <input
+      type="radio"
+      id={id}
+      name={groupName}
+      key={key || groupName + id}
```

Comment:
* I see key quite useless here. There is no array or some other way for React to confuse component instances.
* id is optional so this will print `undefined`
* Inconsistent `groupName + id` and `${groupName}-${id}` between Web and Native.
* Inconsistent id assignment. In Web `groupName + id` is being applied to `key`. In Native it is being applied to `testID`.